### ISSUE TITLE
Sort Order - Tour of Duty

### ISSUE THREAD

**Issue by burgwyn** [18F/State-TalentMAP#450]
Currently the Tour of Duty filters are sorted alphabetically.  These values should be grouped more logically.

<img width="935" alt="screen shot 2017-08-28 at 2 17 56 pm" src="https://user-images.githubusercontent.com/661204/29787173-edb820fa-8bfb-11e7-8cce-0af2a78d3a08.png">

**Comment by afrasier** [comment]
Ordering descending by months makes sense and looks nice. `?ordering=-months`

**Comment by burgwyn** [comment]
Is there a months field?

**Comment by afrasier** [comment]
Yes, it's just called `months`; so you'd do `?ordering=-months` on the tour of duty endpoint

**Comment by esgoodman** [comment]
This might look more like "start date" rather than length of tour, based on research from @krisdelacruz.

**Comment by burgwyn** [comment]
Removing from epic #25

**Comment by burgwyn** [comment]
Removing from epic #786
